developments_build/sql/_function.sql

```diff
@@ -1,5 +1,5 @@
--- occ_translation function
-CREATE OR REPLACE FUNCTION occ_translate(
+-- occ_translation functions
+CREATE OR REPLACE FUNCTION occ_init_translate(
```

Comment:
let's still use 1 function and add a case statement in occ.sql to assign occ_init
```sql
WHEN job_type = 'New Building' THEN 'Empty Site'
```